src/main/java/org/sagebionetworks/web/client/widget/entity/editor/ProvenanceConfigViewImpl.java

```diff
@@ -37,30 +35,26 @@
 	Widget widget;
 
 	@Inject
-	public ProvenanceConfigViewImpl(ProvenanceConfigViewImplUiBinder binder, EntityFinder entityFinder) {
+	public ProvenanceConfigViewImpl(ProvenanceConfigViewImplUiBinder binder, EntityFinder.Builder entityFinderBuilder) {
 		widget = binder.createAndBindUi(this);
-		this.entityFinder = entityFinder;
-		initClickHandlers();
-	}
-
-	private void initClickHandlers() {
-		entityFinderButton.addClickHandler(new ClickHandler() {
-			@Override
-			public void onClick(ClickEvent event) {
-				entityFinder.configure(true, new SelectedHandler<Reference>() {
-					@Override
-					public void onSelected(Reference selected) {
-						if (selected.getTargetId() != null) {
-							appendEntityListValue(selected);
-							entityFinder.hide();
-						} else {
-							showErrorMessage(DisplayConstants.PLEASE_MAKE_SELECTION);
-						}
+
+		entityFinderButton.addClickHandler(event -> entityFinderBuilder
+				.setInitialScope(EntityFinderScope.CURRENT_PROJECT)
+				.setInitialContainer(EntityFinder.InitialContainer.PROJECT)
+				.setHelpMarkdown("Search or Browse Synapse to find an item and display the Provenance Graph within the Wiki page")
+				.setPromptCopy("Find items to insert a Provenance Graph")
+				.setMultiSelect(true)
+				.setSelectableTypes(EntityFilter.ALL)
```

Comment:
Not sure about showing provenance for a Link entity (makes sense for the target of the Link!)